cdap-ml-ext-vertexai/src/main/java/io/cdap/cdap/ml/ext/vertexai/VertexAIConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2024 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.ml.ext.vertexai;
+
+import io.cdap.cdap.common.conf.CConfiguration;
+import io.cdap.cdap.common.conf.Constants;
+
+/**
+ * Configuration class that retrieves Vertex AI-specific configuration parameters
+ * from the given {@link CConfiguration}.
+ */
+public class VertexAIConfiguration {
+
+    private String project;
+    private String location;
+    private String modelName;
+    private String jsonPrompt;
+    private String markDownPrompt;
```

Comment:
How would you differentiate the prompts for different use cases?

Eg. We can prompt to summarize pipeline or summarize wrangler.
Both of them can't use the same variable `markdownPrompt`.